docs/src/DeveloperDocumentation/styleguide.md

```diff
@@ -187,6 +187,9 @@ end
   to understand; it is also more difficult to see all the code at once. Consider
   splitting the function into multiple ones, if it is sensibly possible.
 
+- Every export statement must export exactly one identifier. If multiple
```

Comment:
I guess that's not accurate anymore with the `has_`/`set_` revision. It is also IMHO not necessary; the crucial bit is that every `export` statement should be on a single line, so that one can conveniently grep for exports.

How about this:
```suggestion
- Every export statement must be confined to a single line; the intention is to
  make it easy to use tools like `git grep` to find exports. In general it is
  recommended export exactly one identifier per export statement. Exceptions may
  be made for certain tightly related identifiers, e.g. `is_finite`, `set_is_finite`
  and `has_is_finite` could be put on a single line. In general if multiple
```